index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
+    The controller can also add cues to and remove cues from text tracks.
+
+
+
+All times, time ranges, and durations (such as positon, duration, and
+seekable-time-ranges) used above use a common media-time value (see Appendix A)
+which includes a time scale.  This allows time values which work on different
+time scales to be expressed without loss of precision.  The scale is represented
+in hertz, such as 90000 for 90000hz, a common time scale for video.
+
+
+Remote Playback API {#remote-playback-api}
+------------------------------------------
+
+This section defines how the [[REMOTE-PLAYBACK|Remote Playback API]] uses the
+messages defined in [[#remote-playback-protocol]].
+
+When [[REMOTE-PLAYBACK#the-list-of-available-remote-playback-devices|section
+6.2.1.2]] says "This list contains remote playback devices and is populated
+based on an implementation specific discovery mechanism" and
+[[REMOTE-PLAYBACK#the-list-of-available-remote-playback-devices|section
+6.2.1.4]] says "Retrieve available remote playback devices (using an
+implementation specific mechanism)", the user agent may use the
+mDNS, QUIC, agent-info-request, and remote-playback-availability messages
+defined previously in this spec to discover [=remote playback devices=].  The
+remote-playback-availiability urls must contain the [=availability sources set=].
+
+When
+[[REMOTE-PLAYBACK#establishing-a-connection-with-a-remote-playback-device|section
+6.2.4]] says "Request connection of remote to device. The implementation of this
+step is specific to the user agent." and  "Synchronize the current media element
+state with the remote playback state", the user agent may send the
+remote-playback-start-request message to start remote playback.  The
+remote-playback-start-request urls must contain the [=remote playback source=].
+The current [[REMOTE-PLAYBACK|Remote Playback API]] only allows a single source,
+but the protocol allows for several and future versions of
+[[REMOTE-PLAYBACK|Remote Playback API]] may allow for several.
+
+When
+[[REMOTE-PLAYBACK#establishing-a-connection-with-a-remote-playback-device|section
+6.2.4]] says "The mechanism that is used to connect the user agent with the
+remote playback device and play the remote playback source is an implementation
+choice of the user agent. The connection will likely have to provide a two-way
+messaging abstraction capable of carrying media commands to the remote playback
+device and receiving media playback state in order to keep the media element
+state and remote playback state in sync", the user agent may send
+remote-playback-modify-request messages to change the remote playback state
+based on changes to the local media element and receive
+remote-playback-modify-response and remote-playback-state-event messages to
+change the local media element based on changes to the remote playback state.
+
+<!-- TODO: Have a very descriptive, precise algorithm for what messages to send
```

Comment:
Nit: can we track these TODOs: as issues in the GitHub repository? I think a separate patch converting the TODOs to issues would be handy.

Reply:
Yes, we should have issues for them.